DP3TApp/Logic/Tracing/Reporting/FakePublishManager.swift

```diff
@@ -94,32 +94,48 @@ private class FakePublishOperation: Operation {
     }
 
     override func main() {
-        guard isCancelled == false else { return }
-
-        let startDate = manager.nextScheduledFakeRequestDate
+        var numberOfFakeRequestsDone = 0
+
+        while isCancelled == false,
+            now >= manager.nextScheduledFakeRequestDate {
+            let isFirstReport = numberOfFakeRequestsDone == 0
+
+            // only do request if it was planned to do in the last 48h
+            if now.timeIntervalSince(manager.nextScheduledFakeRequestDate) <= 2 * 24 * 60 * 60 {
+                // add a delay on initial fake report so its not guessable from http traffic if a report was fake or not
+                let group = DispatchGroup()
```

Comment:
Adding the random 20-30 sec delay really doesn't do that much to be honest, especially if the scheduler will always run at the right time when the app is in foreground. But it shouldn't hurt.